apps/next-evans/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "../../node_modules/.pnpm/typescript@5.0.2/node_modules/typescript/lib",
```

Comment:
if these are important, can you move them up to the root level `.vscode/` dir? if not important, nuke them.